frontend/src/modules/Environments/components/EnvironmentRoleAddForm.js

```diff
@@ -91,7 +94,8 @@ export const EnvironmentRoleAddForm = (props) => {
         <Box sx={{ p: 3 }}>
           <Formik
             initialValues={{
-              groupUri: ''
+              groupUri: '',
+              dataallManaged: false
```

Comment:
I would prefer `dataallManaged` to be default as `True` for both new roles via frontend form here and as default in backend (mentioned on an earlier comment by Adriana)

Open for discussion on this - but would think it would help in terms of backwards compatibility for pre-existing roles as well

Reply:
done